src/components/ui/button.tsx

```diff
@@ -9,14 +9,14 @@ const buttonVariants = cva(
   {
     variants: {
       variant: {
-        default: "bg-coral text-dusk hover:bg-coral/90",
```

Comment:
We can change these back later, I think the coral/dusk etc custom colours got overriden somewhere when we restarted tailwind/shadcn, so i made these changes for the buttons' hover states to at least show up